tests/test_push.py

```diff
@@ -185,13 +185,11 @@ def test_succeeded_in_parent_failed_in_current_succeeded_in_child_succeeded_in_g
     assert p[i + 2].get_regressions("label") == {}
 
 
-def test_failure_on_backout(monkeypatch, create_pushes):
+def test_failure_on_backout(create_pushes):
     """
     Tests the scenario where a task succeeded in a parent push, didn't run in the
     push of interest and failed in the push containing the backout of the push of interest.
     """
-    monkeypatch.setattr(HGMO, "is_backout", property(lambda cls: True))
```

Comment:
Sorry, I'm just missing it, but where is `backouts` being populated now?

Reply:
I think I had copied this monkeypatching in almost all tests, even when it was not necessary. It's actually needed only in tests which have 'fixed by commit' classifications, as for the other cases we just rely on `backedoutby`.